Makefile

```diff
@@ -34,41 +31,6 @@ mkdir:
 	mkdir -p ${DIR}/tmp/federation
 
 
-#>>>
-# download all package binaries
-# make bin-all
-
-#<<<
-.PHONY: bin-all
-bin-all: bin-conda
-
-
-#>>>
-# download miniconda package
-# make bin-conda
-
-#<<<
-bin-conda: mkdir
```

Comment:
this target needs to be replicated in init-pipenv, else you don't have a `tmp` dir for the logfile (among other paths):
`bash: /Users/karen/Documents/candig/repos/CanDIGv2/tmp/progress.txt: No such file or directory`

Reply:
Fixed with latest push